extra/html-extra/HtmlExtension.php

```diff
@@ -92,20 +92,20 @@ public static function htmlClasses(...$args): string
     {
         $classes = [];
         foreach ($args as $i => $arg) {
-            if (\is_string($arg)) {
-                $classes[] = $arg;
+            if (\is_string($arg) || $arg instanceof \Stringable) {
```

Comment:
I don't think doing it for all `\Stringable` objects is what we want here.

But having a specific case for `Markup` instanceof might be enough:

```suggestion
            if (\is_string($arg) || $arg instanceof Markup) {
```